docs/root/configuration/http/http_filters/proto_message_logging_filter.rst

```diff
@@ -0,0 +1,8 @@
+.. _config_http_filters_proto_message_logging:
+
+Proto Message Logging
+=====================
+
+* gRPC :ref:`architecture overview <arch_overview_grpc>`
+* This filter should be configured with the type URL ``type.googleapis.com/envoy.extensions.filters.http.proto_message_logging.v3.ProtoMessageLoggingConfig``.
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.proto_message_logging.v3.ProtoMessageLoggingConfig>`.
```

Comment:
Can you add some actual documentation here on what this filter does, why someone would want to use it, and maybe a short config example? Thank you.

/wait

Reply:
Actually, the [config proto](https://github.com/envoyproxy/envoy/blob/06a20dd6ec9f5bd7c6a9e75cc1f54ee0396e7699/api/envoy/extensions/filters/http/proto_message_logging/v3/config.proto) added in #31735 for the filter contains additional details and an example config.

I have verified the doc generation for the v3 API reference using the following command:

```
$ bazel run --//tools/tarball:target=//docs:html //tools/tarball:unpack "$PWD"/generated/docs/
```

IIUC, this will generate the API spec URL under `https://www.envoyproxy.io/docs/envoy/latest/_sources/api-v3/extensions/filters/http/`. This is similar to what we have for the [GFX filter](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/grpc_field_extraction/v3/config.proto).

Please let me know if we want additional details here.

Thank you!

Reply:
Yes please add a least some high level description of what this filter does and why someone would want to use it and then ref links to other places as needed, thanks.

/wait

Reply:
Sure, I have added description, use case and references. Please let me know if it looks good.

Thanks!